File: SDK/PHP-SDK/ApiTubeNewsService.php
<?php

class ApiTubeNewsService
{
	private $client;

	public function __construct(ApiTubeClient $client)
	{
		$this->client = $client;
	}

	public function getArticles($params = [])
	{
		return $this->client->request('/articles', $params);
	}
}

File: SDK/Java-SDK/ApiTubeNewsService.java
// ApiTubeNewsService.java

import java.util.Map;

public class ApiTubeNewsService {
    private ApiTubeClient client;

    public ApiTubeNewsService(ApiTubeClient client) {
        this.client = client;
    }

    public String getArticles(Map<String, String> params) throws Exception {
        return client.request("/articles", params);
    }
}
